### PR TITLE
Cards and Blocks

### PR DESCRIPTION
**Pull request by UladBohdan**
* Proper border radius for Tag Cards within the Block
* Bordering for cards looking white on white background and for cards looking black on black background
Screen Shot 2019-10-19 at 22 14 15
Screen Shot 2019-10-19 at 22 14 28

### CODE REVIEW

**Comment by UladBohdan** [comment]
[to fix border-radius for TagByTopic block](https://trello.com/c/FuJyEtu9/442-to-fix-border-radius-for-tagbytopic-block)

**Comment by UladBohdan** [comment]
[Card Border](https://trello.com/c/tXxQhA63/444-card-border)

File: components/articles/cards/CardWrapper.js [hunk 1]
```diff
@@ -16,7 +17,11 @@ export const SIZES = DEFAULT_SIZES.concat(SQUARE_SIZES, TAG_SIZES, 'auto');
 const CardWrapper = ({ className, size, children, image, color, theme, linkProps }) => (
   <Link className={`card-size-${size}`} {...linkProps}>
     <BgContainer
-      className={cn('wir-card-wrapper', className, { 'theme-light': theme === 'light' })}
+      className={cn('wir-card-wrapper', className, {
+        'theme-light': theme === 'light',
+        'theme-white': colorLooksWhite(color),
+        'theme-black': colorLooksBlack(color),
```
Comment (Drapegnik) [inline components/articles/cards/CardWrapper.js:26]: maybe add comment here, to explain logic / connection to design?

File: utils/ui.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import 'components/common/ui/link.scss';
 
+import every from 'lodash/every';
```
Comment (Drapegnik) [inline utils/ui.js]: `Array.every`

File: utils/ui.js [hunk 3]
```diff
@@ -8,3 +9,17 @@ export const linkCn = ({ className, disabled, dark, active } = {}) =>
     { 'wir-link--disabled': disabled, 'wir-link--theme-dark': dark, 'wir-link--active': active },
     className
   );
+
+// https://stackoverflow.com/questions/5623838/rgb-to-hex-and-hex-to-rgb
+const hexToRgb = hex =>
+  hex
+    .replace(/^#?([a-f\d])([a-f\d])([a-f\d])$/i, (m, r, g, b) => `#${r}${r}${g}${g}${b}${b}`)
```
Comment (Drapegnik) [inline utils/ui.js]: Please extract regex to constant & add link to https://regex101.com

**Review by Drapegnik — CHANGES_REQUESTED** [review]

File: utils/ui.js (hunk 3 above)
Comment (Drapegnik) [reply utils/ui.js]: for example: https://github.com/babajka/babajka-backend/blob/develop/src/services/fibery/utils.js#L14-L15

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**